src/test/MockPolygonBridge.t.sol

```diff
@@ -32,13 +32,12 @@ contract MockPolygonBridgeTest is PRBTest, StdCheats {
     }
 
     /// @notice tests that receiveRoot succeeds if encoded properly
-    function testReceiveRootSucceeds(uint256 newRoot, uint128 supersedeTimestamp) public {
-        bytes memory message =
-            abi.encodeWithSignature("receiveRoot(uint256,uint128)", newRoot, supersedeTimestamp);
+    function testReceiveRootSucceeds(uint256 newRoot) public {
```

Comment:
We should also have tests for failure cases for `receiveRoot`.

Reply:
on it